### ISSUE TITLE
"Power BI Data Analysis Dashboard Integration"

### ISSUE THREAD

**Issue by umursvt** [umursvt/car-price-data-analysis-and-ml-project#2]
Issue Açıklaması:

1. Issue Tanımı:
Power BI kullanarak proje verilerini görselleştirme ve analiz etme adımlarını içerir.
Amaç: Araç fiyat tahminine dayalı sonuçları görselleştirip analiz etmek ve kullanıcılara öngörüler sunmak.
Verilerin anlamlandırılmasına ve daha net öngörüler yapılmasına yardımcı olacak.

2. Yapılacak Adımlar:
Veri Hazırlığı (Data Preparation):

Kaynak: Model eğitimi sırasında kullanılan verileri (özellikler, gerçek fiyatlar, tahmin edilen fiyatlar) içeren bir CSV dosyası veya Excel dosyası Power BI'a aktarılacak.
Temizlik ve Dönüştürme: Gereksiz sütunları kaldırıp, gerekli dönüşümleri uyguladıktan sonra veri setini analiz etmeye uygun hale getir.
Dashboard Tasarımı:

Fiyat Tahmin Performansı Görselleştirmesi:
Model sonuçlarını gerçek fiyatlar ile tahmin edilen fiyatlar arasında kıyaslayan grafikler (örneğin, scatter plot, line graph).
Araç Özelliklerinin Fiyata Etkisi:
Özellikler (kilometre, motor hacmi, marka gibi) ile fiyat arasındaki ilişkiyi gösteren grafikler (bar charts, heatmaps).
Fiyat Dağılımı ve Tahmin Hataları:
Histogram veya boxplot ile fiyat dağılımını ve tahmin hatalarının görselleştirilmesi.
Filtreler ve Etkileşimli Özellikler:

Kullanıcıların veriyi şehir, marka, model, kilometre gibi farklı kategorilere göre filtreleyebilmesi.
Farklı modellerin tahmin başarımlarını karşılaştırabilecek dinamik filtreler.
Veri Güncellemeleri:

Power BI dashboard'unun dinamik olarak veri güncellemelerini alabilmesi için bir plan oluştur (örneğin, yeni model tahmin sonuçları eklendiğinde Power BI verisini güncelleyebilmelisin).
3. Hedefler:
Gerçek ve tahmin edilen fiyatlar arasındaki farkı net olarak göstermek.
Tahmin edilen fiyatların doğruluğunu görselleştirmek ve analiz etmek.
Veri setinde dikkat çeken özellikleri ve bunların fiyata etkilerini interaktif bir şekilde sunmak.
4. Beklenen Sonuçlar:
Power BI dashboard'u, model sonuçlarını görsel olarak anlamlandırma imkanı sunacak.
Projeye ek bir değer katacak, böylece kullanıcılar ve geliştiriciler verileri daha rahat analiz edebilecek.

Issue Checklist:

 Veri setini Power BI ile uyumlu hale getirme.
 Power BI'a veriyi aktarma ve dönüştürme.
 Farklı görselleştirmeleri ve analiz panellerini oluşturma.
 Dinamik filtreler ekleme (şehir, model, yıl gibi).
 Dashboard'u test etme ve kullanıma hazır hale getirme.

**Comment by galip16** [comment]
this seems a big enhancement you can create seperate issues for them or? @umursvt

**Comment by umursvt** [comment]
yeah it seems like that. maybe i should create a 'epic issue' for this steps.
